Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java

```diff
@@ -103,7 +103,7 @@ public void addMultiDocStrucSubmitBtnClick() {
      * This method is invoked if the user clicks on the add single doc struc
      * submit btn command button.
      */
-    public void addSingleDocStrucSubmitBtnClick() {
+    public void addSingleDocStrucSubmit() {
```

Comment:
Is submit needed in the name?

Reply:
You are right, it's probably not needed!